src/lib/core/a11y/interactivity-checker.ts

```diff
@@ -106,6 +177,27 @@ function hasValidTabIndex(element: HTMLElement): boolean {
   return !!(tabIndex && !isNaN(parseInt(tabIndex, 10)));
 }
 
+function getTabIndexValue(element: HTMLElement) {
```

Comment:
Needs function description that explains what this does beyond reading the `tabIndex` property directly.